.github/workflows/ci.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+
+bazel test //...
+readonly exit_code="$?"
+
+case "${exit_code}" in
+  "4")
+    # Status code indicates that the build succeeded but there were no tests to
```

Comment:
could we just add a `build_test` target to avoid the indirection of adding a bash script here? It will likely grow once it exists

Reply:
See other comment about not having dependencies